packages/core/modules/opus_measurement.py

```diff
@@ -352,36 +313,29 @@ def check_for_experiment_change(self) -> None:
         active experiment. To reload an experiment during an active
         macro the macro needs to be stopped first."""
 
-        assert sys.platform == "win32"
-
-        if self.config["opus"]["experiment_path"] != self.current_experiment:
-            if interfaces.StateInterface.read_persistent()["active_opus_macro_id"] == None:
```

Comment:
I don't remember why this was in there. But as far as I can tell, the experiment is loaded into OPUS during the OPUS startup in the morning. After this, it should indeed only matter to startup the macro during measurements and handle the experiment only if it changes.
Let's keep an eye in this during the integration. But it seems AFAIK.

Reply:
I took a while to find it. We have the problem, that Opus answers in cryptic characters (#124), so when starting Opus, we do not get back a Macro ID. Hence, this was always true. I think until now, changing the experiment path while a macro was running did not work.